api/app/Http/Middleware/AuditQueryMiddleware.php

```diff
@@ -36,6 +36,30 @@ public function handle(Request $request, Closure $next)
             );
         }
 
+        // with logging - test query
+        // timing (browser waiting) 16.12, 14.86, 15.78 seconds
+        // logger timing 16.04, 14.80, 15.72 seconds
+
+        // without logging - test query
+        // timing (browser waiting) 15.98, 16.08, 15.98 seconds
+
+        // $start = hrtime(true);
+        // $result = $next($request);
+        // $end = hrtime(true);
+        // $elapsedTime = ($end - $start) / 1000000000;
+
+        // if ($elapsedTime > 0.25) {
+        //     $this->logger->info(
+        //         'Slow query',
+        //         [
+        //             'request' => $request->json()->all(),
+        //             'elapsed_seconds' => $elapsedTime,
+        //         ]
+        //     );
+        // }
+
+        // return $result;
```

Comment:
The method suggested by Peter, appears to work quite fine. As you can see in the comments, tested timing with the logging on vs off and it appear to cause no noticeable slowdown. 
It also gives a nice logging result of the query and timing where it is easy to comprehend what is happening. 

![Screenshot 2024-01-17 172454](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/42d13906-202c-4532-a640-c63be0fabdb0)